Assets/Tilemap/Tiles/Rule Override Tile/Scripts/Editor/PopulateRuleOverrideTileWizard.cs

```diff
@@ -0,0 +1,82 @@
+using UnityEngine;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace UnityEditor {
+	public class PopulateRuleOverideTileWizard : ScriptableWizard {
+
```

Comment:
It is recommended to replace the inspector button with a MenuItem.

### MenuItem Code

```suggestion

        [MenuItem("CONTEXT/RuleOverrideTile/Populate From Sprite Sheet")]
        static void MenuOption(MenuCommand menuCommand)
        {
                PopulateRuleOverideTileWizard.CreateWizard(menuCommand.context as RuleOverrideTile);
        }
        [MenuItem("CONTEXT/RuleOverrideTile/Populate From Sprite Sheet", true)]
        static bool MenuOptionValidation(MenuCommand menuCommand)
        {
                RuleOverrideTile tile = menuCommand.context as RuleOverrideTile;
                return tile.m_Tile && !tile.m_Advanced;
        }

```

### Inspector Button Code

https://github.com/Unity-Technologies/2d-extras/blob/52d0afabc3ab72c3826fb9ed785a8fd4fd4b4407/Assets/Tilemap/Tiles/Rule%20Override%20Tile/Scripts/Editor/RuleOverrideTileEditor.cs#L64-L73